automatically setup and update RHUI repos (#2274)
uyuni-project/uyuni-docs: modules/client-configuration/pages/clients-rh-rhui.adoc

. In the [guimenu]``User`` field, enter the username as specified in the cloud.
. In the [guimenu]``Password`` field, enter the password.
. In the [guimenu]``SSH Private Key`` field, enter the instance key.
. In the [guimenu]``SSH Private Key Passphrase`` field, enter the key passphrase.

[IMPORTANT]
====
Authentication keys must always be in PEM format.
====

If you are not connecting directly to the instance, but via SSH bastion, proceed with <<proc-adding-ssh-bastion-connection-data>>.

Otherwise, continue with <<proc-finishing-rhui-connecting>>.

[[proc-adding-ssh-bastion-connection-data]]
.Procedure: Adding SSH bastion connection data
[role=procedure]
. Navigate to the page section [guimenu]``Bastion SSH connection data``.
. In the [guimenu]``Host`` field, enter the bastion hostname.
. In the [guimenu]``SSH Port`` field, enter the bastion port number.
. In the [guimenu]``User`` field, enter the bastion username.
. In the [guimenu]``Password`` field, enter the bastion password.
. In the [guimenu]``SSH Private Key`` field, enter the bastion key.
. In the [guimenu]``SSH Private Key Passphrase`` field, enter the bastion key passphrase.

> Complete the setup process with with <<proc-finishing-rhui-connecting>>.

Reviewer: ```suggestion
Complete the setup with with <<proc-finishing-rhui-connecting>>.
```

[[proc-finishing-rhui-connecting]]
.Procedure: Finishing {redhat} connecting
[role=procedure]
. To complete adding new {redhat} connection data, click btn:[Create].
. Return to {payg} connection data [guimenu]``Details`` page.
    The updated connection status is displayed on the top section named [guimenu]``Information``.

> . Connection status is shown in [guimenu]``Admin > Setup Wizard > Pay-as-you-go`` screen too.

Reviewer: ```suggestion
. Connection status is shown in [guimenu]``Admin > Setup Wizard > Pay-as-you-go`` screen, too.
```

> . If the authentication data for the instance are correct, the column [guimenu]``Status`` shows "Credentials successfully updated."

Reviewer: ```suggestion
. If the authentication data for the instance is correct, the column [guimenu]``Status`` shows [literal]``Credentials successfully updated.``
```

[IMPORTANT]
====

> If the invalid data are entered at any point, the newly created instance is shown in [guimenu]``Admin > Setup Wizard > {payg}``, with column [guimenu]``Status`` displaying error message.

Reviewer: ```suggestion
If invalid data is entered at any point, the newly created instance is shown in [guimenu]``Admin > Setup Wizard > {payg}``, with column [guimenu]``Status`` displaying an error message.
```

====

As soon as the authentication data is available on the server, repositories were added for all available repositories on the connected instance.
The repositories can be seen in [guimenu]``Software > Manage > Repositories``

[IMPORTANT]
====
A {redhat} connection will create custom repositories which are owned by organization 1 by default.
If a different organization should own the autogenerated repositories, configure [systemitem]``java.rhui_default_org_id`` in ``/etc/rhn/rhn.conf``
====

> This only defines and updates the repositories. If you want to use a repository for a managed client, you need to specify a Software Channel and connect the repositories to it.

Reviewer: ```suggestion
This only defines and updates the repositories.
If you want to use a repository for a managed client, you need to specify a Software Channel and connect the repositories to it.
```
